dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ExclusionFileParser.java

```diff
@@ -17,8 +17,14 @@
 package com.google.cloud.tools.opensource.classpath;
 
 import com.google.common.collect.ImmutableList;
+import com.thaiopensource.util.SinglePropertyMap;
```

Comment:
The package name of this library is strange. This library org.relaxng:jing is from https://relaxng.org/jclark/api/jing/index.html

> You can use Jing's native API.